source/features/table-input.css

```diff
@@ -6,18 +6,33 @@
 	grid-template: repeat(5, 20px) / repeat(5, 20px);
 }
 
-:root .rgh-table-input-cell {
+.rgh-table-input .sentinel { /* 🤷‍♂️ */
+	display: none;
```

Comment:
Did they add this element literally just now while I was writing this PR? I think this effectively forces me to push another release soon because the feature looks  wonky 🤔

Reply:
They dropped it. What a coincidence.